example/src/components/EmployeeSheetImporter.tsx

```diff
@@ -108,9 +108,16 @@ export default function StudentsImporter() {
         EMPLOYEE_SHEET,
         COMPANY_SHEET,
       ]}
-      onDataColumnsMapped={(dataColumns) => {
-        console.log(dataColumns);
-        return dataColumns;
+      onDataColumnsMapped={(sheets) => {
+        const sheet = sheets.find((sheet) => sheet.sheetId === 'companies')!;
+        const seen = new Set();
+        sheet.rows = [...sheet.rows].filter((row: SheetRow) => {
+          // Remove duplicate names, don't validate yet...
```

Comment:
I don't think this bit should validate anything? Shouldn't the package handle validations?